src/YAML.jl

```diff
@@ -15,35 +15,45 @@ include("constructor.jl")
 include("writer.jl") # write Julia dictionaries to YAML files
 
 const _constructor = Union{Nothing, Dict}
-const _dicttype = Union{Type,Function}
+const _dicttype = Union{Type, Function}
 
 # add a dicttype-aware version of construct_mapping to the constructors
 function _patch_constructors(more_constructors::_constructor, dicttype::_dicttype)
     if more_constructors == nothing
-        more_constructors = Dict{String,Function}()
+        more_constructors = Dict{String, Function}()
     else
         more_constructors = copy(more_constructors) # do not change the outside world
     end
     if !haskey(more_constructors, "tag:yaml.org,2002:map")
         more_constructors["tag:yaml.org,2002:map"] = custom_mapping(dicttype) # map to the custom type
-    elseif dicttype != Dict{Any,Any} # only warn if another type has explicitly been set
+    elseif dicttype != Dict{Any, Any} # only warn if another type has explicitly been set
         @warn "dicttype=$dicttype has no effect because more_constructors has the key \"tag:yaml.org,2002:map\""
     end
     return more_constructors
 end
 
-
 load(ts::TokenStream, constructor::Constructor) =
     construct_document(constructor, compose(EventStream(ts)))
 
-load(input::IO, constructor::Constructor) =
-    load(TokenStream(input), constructor)
-
-load(ts::TokenStream, more_constructors::_constructor = nothing, multi_constructors::Dict = Dict(); dicttype::_dicttype = Dict{Any, Any}, constructorType::Function = SafeConstructor) =
-    load(ts, constructorType(_patch_constructors(more_constructors, dicttype), multi_constructors))
-
-load(input::IO, more_constructors::_constructor = nothing, multi_constructors::Dict = Dict(); kwargs...) =
-    load(TokenStream(input), more_constructors, multi_constructors ; kwargs...)
+load(input::IO, constructor::Constructor) = load(TokenStream(input), constructor)
+
+load(
+    ts::TokenStream,
+    more_constructors::_constructor=nothing,
+    multi_constructors::Dict=Dict();
+    dicttype::_dicttype=Dict{Any, Any},
+    constructorType::Function=SafeConstructor,
+) = load(
+    ts,
+    constructorType(_patch_constructors(more_constructors, dicttype), multi_constructors),
```

Comment:
I don't love this formatting

Reply:
Agreed. This is the kind of JuliaFormatter thing that makes me not want to touch a code base.

This particular definition shouldn't have been in short form at all though; there's way too many arguments to make that look good, regardless of formatting.